spec/controllers/admin/cafe_works_controller_spec.rb

```diff
@@ -136,7 +135,7 @@
     end
     it 'create post' do
       lambda {
-        post :setup_create, {commit: 'Spara', cafe_work: attributes_for(:cafe_work), lv_first: 1, lv_last: 1}
+        post :setup_create, {commit: 'Spara', cafe_work: attributes_for(:cafe_work, lv_first: 1, lv_last: 1)}
```

Comment:
Line is too long. [109/100]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.